.gitlab-ci.yml

```diff
@@ -37,6 +37,28 @@ trigger_child_pipeline:
     strategy: depend
   needs:
     - generate_matrix
+  
+check-big-regressions:
+  stage: benchmarks
+  needs:
+    - job: trigger_child_pipeline
+      artifacts: true
+  when: on_success
+  tags: 
+    - "arch:amd64"
+  image: $BASE_CI_IMAGE
+  script:
+    - |
+      ls -la
+      echo "+++++++++"
+      pwd
+      echo "+++++++++"
+      ls -la /
+    - git clone --branch dario.castane/APMSP-2045/perf-quality-gate https://gitlab-ci-token:${CI_JOB_TOKEN}@gitlab.ddbuild.io/DataDog/benchmarking-platform platform && cd platform
+    - bp-runner bp-runner.fail-on-regression.yml --debug
```

Comment:
How will the UX look like? I would love to see a GH comment as we see in the current benchmark checks.

Reply:
The expected UX is the failing Ci job blocking the PR. The current benchmark GH comment isn't going away with this new implementation, it'll be generated as usual.

This is more a standardization effort across tracers to have similar pipelines.